CollAction/Views/Projects/Create.cshtml

```diff
@@ -21,17 +21,17 @@
                     </div>
```

Comment:
I don't think these changes are part of CA-292. The rest looks fine, but this should probably be reverted.